### PR TITLE
Fix poo #28648

### PR DESCRIPTION
**Pull request by drpaneas**
Add an exception for QAM CaaSP workflow in respect to autoyast.
This is required for the stability of our scenarios. The `generic`
autoyast test is using _too many_ needle tags, thus our systems
are prone to miss the `bios-boot` tag/needle when we deploy a
*10-node* CaaSP cluster.

The reason behind this is that we are using autoyast just to bring
the workers up to the defined state. Any error that happens in
between, is not really important to capture it _on time_. In any
case, if something breaks during the autoyast installation, we will
encounter it anyway in the end of the installation.

Simply put, risking the stability of our tests in order to save
a couple of minutes it's something we would like to avoid.

As a result, this PR removes the ability of consistantly scanning
for errors during the autoyast installation, but it keeps looking
for the only needle we care specifically.

- Related ticket: https://progress.opensuse.org/issues/28648
- Needles: not needed
- Verification run: http://skyrim.qam.suse.de/tests/overview?distri=caasp&version=2.0&build=4&groupid=98

### CODE REVIEW

File: tests/autoyast/installation.pm [hunk 1]
```diff
@@ -105,6 +105,13 @@ sub run {
     my $timer      = 0;    # Prevent endless loop
 
     mouse_hide(1);
+
+    # QAM CaaSP specific
+    if (is_caasp && check_var('FLAVOR', 'CaaSP-DVD-Incidents')) {
+        assert_screen('bios-boot', 1200);
```
Comment (rwx788) [inline tests/autoyast/installation.pm]: I guess it will be easier to adjust needles array for this scenario, and just put one needle there and using same code as before. I also not sure if it's really what you want to see, as you can ignore warning/errors in the profile. This loop helps to fail test before it times out in 2 hours, which is good from many perspectives. So I would leave at least 'bios-boot' and 'linuxrc-install-fail'. You may also introduce needle which is there when system is already booted, then we don't care that we have missed bios-boot

Comment (drpaneas) [reply tests/autoyast/installation.pm]: yeah, doesn't make much sense. Can you reject this PR?
